### PR TITLE
Feature/task 774 swt bot to check that 

### LINKED ISSUE #774: SWTBot to check that licensing info is present in all our features and plugins

### PR DESCRIPTION
**Pull request by andriikovalov-dlr**
Added a test that checks that licensing information is present in all features and plugins, and vendor is correctly set
Fixed issues revealed with this test

Closes #774

### CODE REVIEW

**Review by JAmmermann-DLR — APPROVED** [review]

File: de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/LicenseTest.java [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+			String pluginId = row.getText(COLUMN_PLUGIN_ID);
+			if (pluginId.startsWith(VIRSAT_PACKAGE_PREFIX)) {
+				bot.table().select(i);
+				assertEquals("Correct provider for plugin " + pluginId, EXPECTED_PROVIDER, row.getText(1));
+				assertTrue("Licensing information is set for plugin " + pluginId, bot.button("Legal Info").isEnabled());
+			}
+		}
+		
+		bot.button("Close").click();
+		bot.button("Close").click();
```
Comment (SaMuellerDLR) [inline de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/LicenseTest.java:76]: Some comment on why we need to press Close twice would be good.
I guess there are two dialogues open at this stage?

**Review by SaMuellerDLR — CHANGES_REQUESTED** [review]
Wow this is awesome. Didnt think you could test this with SWTBot! 👍 

Only a minor comment request.

File: de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/LicenseTest.java (hunk 1 above)
Comment (andriikovalov-dlr) [reply de.dlr.sc.virsat.swtbot.test/src/de/dlr/sc/virsat/swtbot/test/LicenseTest.java:77]: Yes, for installation details and about window, added a comment to clarify.

**Review by SaMuellerDLR — APPROVED** [review]
Approved!

**Pull request merged**